glusterd2/bricksplanner/planner.go

```diff
@@ -137,6 +141,9 @@ func getBricksLayout(req *api.VolCreateReq) ([]api.SubvolReq, error) {
 				TpMetadataSize: deviceutils.GetPoolMetadataSize(eachBrickTpSize),
 				FsType:         "xfs",
 				MntOpts:        "rw,inode64,noatime,nouuid",
+				Metadata: map[string]string{
```

Comment:
This is still adding Metadata to request structure(All Metadata key starting with `_` need to be restricted via APIs). Add this directly to volinfo in the handler